client/go/dogma/dogma.go

```diff
@@ -0,0 +1,332 @@
+// Copyright 2018 LINE Corporation
+//
+// LINE Corporation licenses this file to you under the Apache License,
+// version 2.0 (the "License"); you may not use this file except in compliance
+// with the License. You may obtain a copy of the License at:
+//
+//   https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations
+// under the License.
+
+package dogma
+
+import (
+	"bytes"
+	"context"
+	"encoding/json"
+	"fmt"
+	"golang.org/x/oauth2/clientcredentials"
+	"io"
+	"io/ioutil"
+	"net/http"
+	"net/url"
+	"strconv"
+	"strings"
+)
+
+const (
+	DefaultScheme     = "http"
+	DefaultHostName   = "localhost"
+	DefaultPathPrefix = "api/v1/"
+	PathPrefixV0      = "api/v0/"
+	DefaultBaseURL    = DefaultScheme + "://" + DefaultHostName + ":36462/"
+
+	defaultPort         = 36462
```

Comment:
I actually meant making all constants private, except `DefaultPort`. :-)

Reply:
Oops~~ let me fix this~